WDL/CLI.py

```diff
@@ -283,12 +283,27 @@ def fill_cromwell_subparser(subparsers):
         action="append",
         help="explicitly set an array input to the empty array",
     )
+    cromwell_parser.add_argument(
+        "-r",
+        "--jar",
+        metavar="jarfile",
+        dest="jarfile",
+        type=str,
+        help="Cromwell jarfile filename/URI",
+    )
+    cromwell_parser.add_argument(
+        "-c",
+        "--config",
+        metavar="CONFIG",
+        dest="config",
+        type=str,
+        help="Cromwell backend configuration filename/URI",
```

Comment:
This is a JSON file, right?

Suggest metavar="CONFIG.json", help="Cromwell backend configuration JSON filename (also set by CROMWELL_CONFIG environment variable)"